### PR TITLE
Various sound fixes

### PR DESCRIPTION
**Pull request by NPC1314**
## About The Pull Request

MUST HAVE TESTMERGE BEFORE FULLMERGE, IMPORTANT
This due to falloff edits being finicky when testing solo on testserve vs on live so it might be annoying.

( Adds gotemilk - removed this because its done in the animal butchery/mob file cleanup, no need to have it here with sounds )

Some misc afterbirth from the audio project. To clairfy, project Normalize, where LUFS was organized, focus on getting various soundscape bits in better shape. Came as far as footsteps, most ambience tracks & music but more tweaking  needed and more sections could do with curating, such as emotes that only got partially fixed, various SFX, etc etc. Look at the devlog from april or whatever for more context. 

* Couple ambient music tracks werent normalized
* Combat music needed a little more oomph so got 20%  boost
* Couple sound effects normalized via code
Reminder the default music volume is 50 which the soundscape more or less is designed for, for those who want more loud music all the time just crank it up to 100 in your options.
Couple of unused audio files moved to the right folder.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by NPC1314** [comment]
> Does this fix the quiet instruments problem? Right now someone right next to you could be playing a tune, and you can barely hear it.

Wasnt aware of the issue but yeah, should help now, tweaking fallofs a little, Ill keep an eye and tweak it more until its just right.

**Review by Optimism333 — COMMENTED** [review]
Hoping that gote milk will be a good prerequisite for some additional cheese variety that I plan to add.

File: code/game/objects/items.dm [hunk 1]
```diff
@@ -644,10 +647,14 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 			A.Grant(user)
 	item_flags |= IN_INVENTORY
 	if(!initial)
-		if(equip_sound &&(slot_flags & slotdefine2slotbit(slot)))
+		var/slotbit = slotdefine2slotbit(slot)
+		if(slot == SLOT_HANDS) // for some reason doesnt work. Replace with any other slot it does. Im stumped.
```
Comment (ThePainkiller) [inline code/game/objects/items.dm:651]: Shouldn't this be `slotbit == SLOT_HANDS` ?

Comment (EvenInDeathIStillServe) [reply code/game/objects/items.dm:651]: This should be `slot == ITEM_SLOT_HANDS`, actually! Picking up an item called `equipped()` with the slot argument of `ITEM_SLOT_HANDS`. Using slotbit would give the wrong number!

Comment (ThePainkiller) [reply code/game/objects/items.dm:651]: I actually found a better fix and pushed it on my PR last night

**Comment by NPC1314** [comment]
Pending clearing up a procedure thing on hold

**Comment by NPC1314** [comment]
All good for merge

**Review by ThePainkiller — APPROVED** [review]
Let's get this sound normalization merged already, if anything else shows up let players report it so we can modify their volume.

**Pull request merged**